seedfarmer/commands/_deployment_commands.py

```diff
@@ -56,21 +57,38 @@
 _logger: logging.Logger = logging.getLogger(__name__)
 
 
-def _process_module_path(module: ModuleManifest) -> None:
+def _process_git_module_path(module: ModuleManifest) -> None:
     working_dir, module_directory, commit_hash = sf_git.clone_module_repo(module.path)
     module.set_local_path(os.path.join(working_dir, module_directory))
     module.commit_hash = commit_hash if commit_hash else None
 
 
-def _process_data_files(data_files: List[DataFile], module_name: str, group_name: str) -> None:
+def _process_archive_path(module: ModuleManifest, secret_name: Optional[str] = None) -> None:
```

Comment:
why was the data path removed?  Was this tested with a module using a datapath...such as:

```
name: mwaa
path: modules/orchestration/mwaa
dataFiles:
  - filePath: archive::https://github.com/awslabs/idf-modules.zip?module=data/mwaa/requirements/requirements-emr-serverless.txt
```
Just want to confirm that this was addressed

Reply:
I tested it with a datapath, it worked. It seems like git diff is just displaying this change in a strange way. I didn't replace `_process_data_files` with `_process_archive_path`. The `_process_data_files` function is still there, it's just after the definition of `_process_archive_path`,